.github/workflows/release.yaml

```diff
@@ -80,7 +80,7 @@ jobs:
               cd $SERVICE || exit 1
               autonomy build-image || exit 1
               autonomy build-image --version $VERSION || exit 1
-
+              docker tag oar-impact_evaluator:$VERSION oar-$SERVICE:$VERSION
```

Comment:
I think it is `$AUTHOR/oar-impact_evaluator:$VERSION`, right?
Also, please make sure that `docker` is available here, otherwise tag in the `Docker Push` section below.

Reply:
Right! Updated. I think docker is available (at least other jobs use it without specifying extra deps)